### ISSUE TITLE
Neon Wallet 0.0.7 & 0.0.6 Troubleshooting

### ISSUE THREAD

**Issue by Tristan1986** [CityOfZion/neon-wallet#494]
Hi there,

A few weeks ago I installed my neon wallet 0.0.7 correctly @ Windows 10. Everything worked fine! Yesterday I needed version 0.0.6 and installed it correctly @ the same pc. After this, I didn't needed 0.0.6 anymore and installed 0.0.7 again. 

Problem:

The setup is opening --> ''Installing, please wait...'' and after this nothing happened. I can't find the neon wallet anywhere and there is no shortcut neither anymore. Same problem with other versions now.

I just used the same setupfiles (downloaded from here) but it doesn't work. Does someone know how to solve this problem or to tell me what I'm doing wrong?

Thanks!

**Comment by stevesbrain** [comment]
What about 0.0.9? It's the latest version :)

**Comment by Tristan1986** [comment]
Yeah well, there was some Windows registry update needed, thats the reason why 0.9 bugged:

https://www.windows10forums.com/threads/ms-setting-display-and-ms-personalization-background-problem.1247/

It's fixed now!
